spec/support/shared_examples/ial2_consent.rb

```diff
@@ -1,11 +1,7 @@
 shared_examples 'ial2 consent with js' do
-  it 'does not allow the user to continue without checking the checkbox' do
```

Comment:
Could we replace this with a test that confirms the notice is shown if they try to click Continue without giving consent?